build.proj

```diff
@@ -81,18 +87,21 @@
       <TestDependencyListFile Include="$(BinDir)/TestDependencies/*.dependencylist.txt" />
     </ItemGroup>
 
+    <PropertyGroup>
+      <TestArchiveDir>$(TestWorkingDir)$(OSPlatformConfig)/archive/tests/</TestArchiveDir>
+      <TestArchiveDir Condition="'$(TestTFM)' != ''">$(TestArchiveDir)$(TestTFM)/</TestArchiveDir>
```

Comment:
We aren't using TestTFM any longer. Is there any reason we cannot just have these in one root bin/archive folder and make the name of the zip the unique part?